backend/cmd/migrate/migrate.go

```diff
@@ -13,13 +13,12 @@ import (
 	"github.com/golang-migrate/migrate/v4/database/postgres"
 	_ "github.com/golang-migrate/migrate/v4/source/file"
 	_ "github.com/lib/pq"
-	"github.com/uber-go/tally/v4"
-	"go.uber.org/zap"
-
 	gatewayv1 "github.com/lyft/clutch/backend/api/config/gateway/v1"
 	postgresv1 "github.com/lyft/clutch/backend/api/config/service/db/postgres/v1"
 	"github.com/lyft/clutch/backend/gateway"
 	clutchpg "github.com/lyft/clutch/backend/service/db/postgres"
+	"github.com/uber-go/tally/v4"
+	"go.uber.org/zap"
```

Comment:
why are imports no longer grouped properly?

Reply:
Hmm, I usually see folks using only two groups (stdlib and non-stdlib), but I can adjust it to make it Python-style again. Let me do that.

Reply:
Actually, `gci` is making this change. So if we want to revert it we'd need to remove gci from `.golangci.yml`.